tests/jerry/es.next/arguments.js

```diff
@@ -217,3 +217,8 @@ function f22 (arguments, [a = arguments]) {
   assert(arguments === 3.1);
 }
 f22(3.1, []);
+
+function f23(arguments, eval = () => eval()) {
+  assert(arguments === 3.1);
```

Comment:
I would prefer an undefined comparison to prove that the arguments object has never been created (and not overwritten later by the number argument)

Reply:
updated.